terracumber_config/tf_files/SUSEManager-4.1-build-validation.tf

```diff
@@ -197,7 +197,7 @@ module "base_new_sle" {
   name_prefix = "suma-bv-41-"
   use_avahi   = false
   domain      = "mgr.prv.suse.net"
-  images      = [ "sles15o", "sles15sp1o", "sles15sp2o", "sles15sp3o" ]
+  images      = [ "sles15o", "sles15sp1o", "sles15sp2o", "sles15sp3o", "sles15sp4o" ]
```

Comment:
Adding this to 4.1 doesn't make sense at all.
Once SUMA 4.3 is released, SUMA 4.1 will be EoL.
Once SLES15SP4 is released, we release SUMA 4.3 :)

Reply:
Let's cover only 4.2 and 4.3

Reply:
oops... 

ok, i'll remove it here, but keep it in infra

Reply:
Done